metadata.json

```diff
@@ -39,14 +39,15 @@
       "operatingsystemrelease": [
         "5",
         "6",
-        "7"
+        "7",
+        8
```

Comment:
OracleLinux 8 exists and this should work fine with it.  On the other hand, there is no Scientific Linux 8.

Reply:
the `8` needs to be quoted?